pkg/cli/admin/inspect/namespace.go

```diff
@@ -65,10 +65,10 @@ func (o *InspectOptions) gatherNamespaceData(baseDir, namespace string) error {
 		resourcesToStore[gvr] = list
 	}
 
-	log.Printf("    Gathering pod data for namespace %q...\n", namespace)
+	klog.V(1).Infof("    Gathering pod data for namespace %q...\n", namespace)
```

Comment:
these messages exist because the process of collection takes a long time and we needed status indicators.